app/src/main/java/org/greenstand/android/TreeTracker/models/messages/database/MessagesDAO.kt

```diff
@@ -58,6 +58,12 @@ interface MessagesDAO {
     @Query("UPDATE messages SET is_read = 1 WHERE id IN (:id)")
     suspend fun markMessageAsRead(id: List<String>)
 
+    @Query("SELECT COUNT(*) FROM messages WHERE is_read = 0")
+    suspend fun getTotalUnreadMessagesCount(): Int
+
+    @Query("SELECT COUNT(*) FROM messages WHERE wallet = (:wallet) AND is_read = 0")
+    suspend fun getTotalUnreadMessagesPerUserCount(wallet: String): Int
```

Comment:
This name seems longer than it needs to be. Can we try something like 'getUnreadMessageCountForWallet'?